test/sys/test_sockopt.rs

```diff
@@ -17,28 +14,24 @@ fn test_so_buf() {
     assert!(actual >= bufsize);
 }
 
+// The CI doesn't supported getsockopt and setsockopt on emulated processors
```

Comment:
You should mention in your comment that it's believed to be a QEMU issue.

Reply:
> > CI fails but actually the code is OK. I tried CI on a full emulated MIPS or PowerPC system and the test run well. I think that the CI fails because only the binary is run on a emulated processor but the kernel is still the same than the host.
> 
> QEMU does emulate the full system. It's not just a binary translator. However, it's got a lot of bugs. What did you use for a "full emulated MIPS or PowerPC system"?

I installed a full MIPS debian in a virtual machine with `qemu-system-mips` and the run the test inside it. The CI just cross compile the binary and run it with `qemu-mips` so the syscalls are not made by a native kernel.